bindings/nodejs/tests/utils.mjs

```diff
@@ -54,3 +54,23 @@ export function loadConfigFromEnv(scheme) {
       .map(([key, value]) => [key.replace(prefix, ''), value]),
   )
 }
+
+// A readable stream that reads from a buffer
+export class BufferStream extends Readable {
```

Comment:
We shouldn't introduce a non-extendable public API like this. We should allow users to configure the buffer while creating `Reader` and `Writer`.

For example:

```node
let  = Writable.from(op.writerSync({ buffer : 4*1024*1024}))
```

Reply:
This is not a public API. It's for testing.

Reply:
> This is not a public API. It's for testing.

It's better not to have `testing` only API

Reply:
Removed.
In fact, this is not an API. It's a testing utility class. 
But there are things that can replace it. It's better to remove it.